packages/@aws-cdk/core/lib/size.ts

```diff
@@ -178,6 +201,7 @@ export interface SizeConversionOptions {
 
 class StorageUnit {
   public static readonly Kibibytes = new StorageUnit('kibibytes', 1);
+  public static readonly Bytes = new StorageUnit('bytes', 1 / 1024);
   public static readonly Mebibytes = new StorageUnit('mebibytes', 1024);
   public static readonly Gibibytes = new StorageUnit('gibibytes', 1024 * 1024);
   public static readonly Tebibytes = new StorageUnit('tebibytes', 1024 * 1024 * 1024);
```

Comment:
The `1 / 1024` makes me uneasy. Computers aren't good at representing fractional numbers and getting math with them correct.

Can we shift all fractions up by `1024` ? Make `bytes = 1`, `kibibytes = 1024`, etc ?